server/src/cache/macros.rs

```diff
@@ -0,0 +1,4 @@
+//! log macro's for cache logging
+
+use lib_common::log_macros;
+log_macros!("cache", "app::cache");
```

Comment:
should be `backend::cache` instead of `app::cache`

Reply:
https://github.com/aetheric-oss/svc-cargo/pull/43/commits/5eccc2bc4b4335866a129e31c449d470990edf83